.github/workflows/image.yaml

```diff
@@ -303,19 +303,20 @@ jobs:
           - flavor: "opensuse"
             flavorRelease: "leap-15.5"
             label: "provider-qrcode-install"
-          # Requires at least 1 release with the newer repositories to work as this test checks the release list
-          #- flavor: "opensuse"
-          #  flavorRelease: "leap-15.5"
-          #  label: "provider-upgrade"
           - flavor: "opensuse"
             flavorRelease: "leap-15.5"
-            label: "provider-decentralized-k8s"
+            label: "provider-upgrade"
+          # no point of running this on CI if it always fails
+          # - flavor: "opensuse"
+          #   flavorRelease: "leap-15.5"
+          #   label: "provider-decentralized-k8s"
```

Comment:
Ok I get this one might be controversial but seriously, if they always fail what is the point of running them? The real solution is to fix them but that can be done on a PR I guess so I do think we should remove them from master, it just makes it harder to track when something actually broke on master

Reply:
I'm fine to comment them out and open a ticket to fix them (so we don't forget it). Preferably a high prio one. They've been broken for too long.